docs/modules/ROOT/pages/features.adoc

```diff
@@ -0,0 +1,338 @@
+
+The attribute settings are applied in the above order of precedence, with Query paramaters being applied last.
+
+[#url-query-parameters]
+== URL query parameters
+
+You may use URL query parameters to change how the extension previews AsciiDoc. You can set:
+
+* attribute values with `?attribute=<value>`
+* stylesheet used with `?stylesheet=<stylesheet name>` 
+
+*Where to put the query parameters*
+
+Query parameters are indicated by the first (`?`) in the URL and are terminated by the character (`#`) or the end of the URL.
+
+TIP: If setting attributes or the stylesheet via the URL is not working, check that the Query Parameters are before any (`#`) character in the URL.
+
+*Add the first query parameter (`?`)*
+
+For example: To condense a table of contents to one level its current setting, add `?toclevels=1` to URL, after the document name, and before any (`#`) character.
+
+.URL query parameters
+[,url,subs=+quotes]
+----
+https://example.com/page.adoc**?toclevels=1**
+----
+
+*Add a second and more parameters (`&`)*
+
+To add one more parameter, add an ampersand (`&`) to the end of the last query parameter, followed by the next attribute setting.
+
+It will look like this:
+
+URL query parameters
+[,url,subs=+quotes]
+----
+https://example.com/page.adoc**?toclevels=1&stylesheet=maker**
+----
+*Example*
+
+You have a document `ex-query.adoc` that uses two attributes `\{first}` and `\{last}` to say "Hello \{first} \{last}!" in the document title.
+
+.ex-query.adoc
+[subs="-attributes",indent=0]
+----
+= Hello {first} {last}!
+
+Query parameters example
+----
+
+TIP: Download the above document from xref:attachment$ex-query.adoc[ex-query.adoc]
+
+Then, to send a first and last name to the document via the URL, you would add `?first=Irma&last=Writer` to the URL directly after the `ex-query.adoc` and before any (`#`) character.
+
+.URL query parameters
+[,url,subs=+quotes]
+----
+file:///C:/Users/User/Documents/ex-query.adoc**?first=Irma&last=Writer**
+----
+
+
+You should see the following result:
+
+[indent=0]
+====
+[discrete]
+= Hello Irma Writer!
+
+Query parameters example
+====
+
+== Change Themes and Stylesheets
+
+Change the stylesheet, CSS, or both, that are used to render HTML5 from the AsciiDoc.
+For example, you may prefer a darker or a more compact presentation style.
+
+The Theme/Stylesheet can be changed for:
+
+* all files previewed via the extension xref:theme-stylesheet-option.adoc[]
+
+* the current file with <<url-query-parameters>>.
+
+== View AsciiDocs locally and remote
+
+View AsciiDoc files from different locations.
+
+The extension allows you to view AsciiDoc files from your local file system and from the web:
+
+* Local URLs start with `file://`
+* Remote URLs start with `https://`
++
+This includes hosted Git platforms
++
+TIP: *Raw Mode*: To view AsciiDoc files using the {project-name} and not the native AsciiDoc rendering if offered, use the provider's *raw* viewing mode for the file.
```

Comment:
I think we should explicitly mention GitLab and GitHub. We can even add a link to: https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#viewing-or-copying-the-raw-file-content

GitLab does not have a documentation page on this feature :(

```suggestion
TIP: Use _Raw Mode_ to view AsciiDoc files using the {project-name} and not native rendering provided by developer platforms such as GitHub or GitLab.
```